.travis.yml

```diff
@@ -24,7 +24,7 @@ before_script:
   - chmod +x checkpatch.pl
   - clang-format() { clang-format-4.0 $@; }
   - export -f clang-format
-  - export CHECKPATCH_OPTIONS="--terse --no-tree --max-line-length=80 --ignore ARRAY_SIZE --ignore VOLATILE"
+  - export CHECKPATCH_OPTIONS="--terse --no-tree --max-line-length=80 --ignore ARRAY_SIZE, VOLATILE, LEADING_SPACE"
```

Comment:
I think the space is not supposed to be there. According to the `checkpatch.pl` documentation there should be none (this is a bash command, so it makes sense to avoid spaces).

Also, if you carefully check the Travis log, you'll see that the `checkpatch.pl` "failed" (`VOLATILE,: No such file or directory`). Don't know why it did pass instead of returning an error... :confused:

Reply:
Ok, spaces deleted and I will check the next Travis log to confirm that the error comes from that.